cmd/devstream/develop.go

```diff
@@ -19,31 +15,44 @@ var (
 var developCMD = &cobra.Command{
 	Use:   "develop",
 	Short: "Develop is used for develop a new plugin",
-	Long: `Develop is used for develop a new plugin.
+}
+
+var developCreateCMD = &cobra.Command{
+	Use:   "create-plugin",
+	Short: "Create a new plugin",
+	Long: `Create-plugin is used for creating a new plugin.
+Exampls:
+	dtm develop create-plugin --name=YOUR-PLUGIN-NAME`,
+	Run: developCreateCMDFunc,
+}
+
+var developValidateCMD = &cobra.Command{
+	Use:   "validate-plugin",
+	Short: "Validate a plugin",
+	Long: `Validate-plugin is used for validating an existing plugin or all plugins.
 Examples:
-  dtm develop create-plugin --name=YOUR-PLUGIN-NAME,
-  dtm develop validate-plugin --name=YOUR-PLUGIN-NAME`,
-	Run: options.WithValidators(developCMDFunc, options.ArgsCountEqual(1), validateDevelopArgs),
+	dtm develop validate-plugin --name=YOUR-PLUGIN-NAME,
```

Comment:
validate-plugin -> create-plugin

Reply:
Here I put the example information into two different subcommands.

Reply:
Sorry, my mistake.